src/routes/_api/timelines.js

```diff
@@ -66,7 +66,7 @@ export async function getTimeline (instanceName, accessToken, timeline, maxId, s
   }
 
   if (timeline === 'notifications/mentions') {
-    params.exclude_types = ['follow', 'favourite', 'reblog', 'poll', 'admin.sign_up']
+    params.exclude_types = ['follow', 'favourite', 'reblog', 'poll', 'admin.sign_up', 'update', 'follow_request']
```

Comment:
Realized we forgot to add `'update'` here. There are so many notification types now (signup, update, follow request, report), this system is becoming untenable.